src/styles/blocks/section1.scss

```diff
@@ -0,0 +1,37 @@
+.section_1 {
+  margin: 0;
+  padding: 32px 16px;
+  height: 134px;
+  width: 166px;
+  text-align: center;
+}
```

Comment:
The BEM naming convention suggests that elements should be denoted by two underscores (__), and modifiers should be denoted by two hyphens (--). The class name '.section_1' does not follow this convention. Consider renaming it to '.section-1' if it is a block or '.section__element' if 'section_1' is an element of the 'section' block.